packages/esm-patient-notes-app/src/types/index.ts

```diff
@@ -32,29 +33,33 @@ export interface PatientNote {
 }
 
 export interface DiagnosisData {
-  word: null;
-  conceptName: {
-    id: number;
+  uuid: string;
+  display: string;
+  name: {
```

Comment:
Do we care about the name property at all? We only really need to include the properties we care about (we should use `display`).